homeassistant/helpers/script.py

```diff
@@ -254,7 +254,13 @@ async def async_validate_action_config(
         platform = await device_automation.async_get_device_automation_platform(
             hass, config[CONF_DOMAIN], "action"
         )
-        config = platform.ACTION_SCHEMA(config)  # type: ignore
+
+        # XXX
+        # problem is that this config is already run through the base schema,
+        # which converts the template strings to Template, which causes this to fail
+        # when it extends the base, because Template isn't able to be turned into a Template
+
+        # config = platform.ACTION_SCHEMA(config)  # type: ignore
```

Comment:
unsure how to get this to validate - this is the 2nd pass of this particular config being validated, is there a way to make it ignore the base values' contributions (changing strings to Template)?